.NET/Microsoft.Recognizers.Text.NumberWithUnit/Extractors/NumberWithUnitExtractor.cs

```diff
@@ -128,6 +129,34 @@ public List<ExtractResult> Extract(string source)
                     }
 
                     mappingPrefix.TryGetValue(start, out PrefixUnitResult prefixUnit);
+
+                    // For currency unit, add the prefixUnit with number here.
+                    if (prefixUnit != null && !prefixMatched && this.config.ExtractType.Contains("currency"))

```

Comment:
What do you mean add it "here"? Can you give some example?
Any why would this code only run for Currency?